client/src/index.jsx

```diff
@@ -31,12 +31,17 @@ const ErrorBoundary = bugsnagClient.getPlugin('react')
 
 FirebaseApp.auth().onAuthStateChanged((user) => {
   if (user) {
-    // User is signed in; render app
-    ReactDOM.render(
-      <ErrorBoundary>
-        <AppRouter user={user}/>
-      </ErrorBoundary>,
-      document.getElementById('root'))
+    // User is signed in; fetch token and render app
+    FirebaseApp.auth().currentUser.getIdTokenResult()
+      .then(token => {
+
+        ReactDOM.render(
+          <ErrorBoundary>
+            <AppRouter user={user} isAdmin={token.claims.admin || false}/>
+          </ErrorBoundary>,
+          document.getElementById('root'))
+      })
+      .catch(console.error) // FIXME handle errors
```

Comment:
If you manage to run the new logic inside `<ErrorBoundary>`  errors  will show up on BugSnag